### ISSUE TITLE
How many epochs does DeepSpeech2 need to converge on LibriSpeech

### ISSUE THREAD

**Issue by Some-random** [samsungsds-rnd/deepspeech.mxnet#6]
I read the [issue]( https://github.com/dmlc/mxnet/issues/6418 ) regarding the performance of DeepSpeech2 and noticed the CER result reported by @Soonhwan-Kwon is 0.15648 at epoch 3. 

It seems really promising so I'm trying to reproduce the result. But right now I'm at epoch 5 and my validation CER (vali-clean and val-other) is still 0.3122.... So I'm wondering whether I did anything wrong or was that the intended result.

Also, the test other CER on LibriSpeech reported in [DeepSpeech2 paper](https://arxiv.org/pdf/1512.02595.pdf) was 0.1325. Have you guys ever come close to this number? And if so, how many epochs do you need to get there?

Thanks in advance!

**Comment by Soonhwan-Kwon** [comment]
I'm middle of training bucketing model, but cer was 0.177740, 0.144390,0.126324,0.1210941 from Epoch 1 to 4 for test-clean dataset (There is no Epoch 0 result because at Epoch 0 and about 80% of data was through I accidentally powered off the server).
And I have no accuracy result for val-clean and val-other or test other yet, but I'll post them when the model is converged.

**Comment by Soonhwan-Kwon** [comment]
Can you post your cfg file or what is your training data?

**Comment by Some-random** [comment]
@Soonhwan-Kwon Thanks for your response! I checked my cfg again and realized I'm only using three bigru layers. I guess that's the reason why I'm not converging as fast as expected...

**Comment by Soonhwan-Kwon** [comment]
It's a relief you found the cause. Above accuracy was from the 5 bi-gru layers but if you have sufficient equipment please try 7-layers architecture. Thank you for your feed back.
